rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -544,7 +555,18 @@ void log_frame::UpdateUI()
 				// clear selection or else it will get colorized as well
 				text_cursor.clearSelection();
 
-				const QString tty_text = QString::fromStdString(buf_line);
+				// Strip ANSI color code
+				QRegularExpression ansi_color_code("\\[\\d+;\\d+(;\\d+)?m|\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]");
```

Comment:
please put this into the correct scope.
Also, it can be static const or even constexpr

Reply:
fixed